src/x11_window.c

```diff
@@ -2450,6 +2450,10 @@ int _glfwPlatformWindowMaximized(_GLFWwindow* window)
     Atom* states;
     unsigned long i;
     GLFWbool maximized = GLFW_FALSE;
+
+    if (!_glfw.x11.NET_WM_STATE)
+      return maximized;
```

Comment:
You need to use four spaces for indentation here.

```suggestion
        return maximized;
```

Reply:
Good catch, I've just fixed this.